Core/Core/Extensions/UIViewExtensions.swift

```diff
@@ -126,4 +126,17 @@ extension UIView {
             pin(inside: superview, leading: nil, trailing: nil, top: 0, bottom: 0)
         }
     }
+
+    public func disableIfOnline() {
+        self.alpha = 0.5
+        self.gestureRecognizers?.removeAll()
```

Comment:
If we remove all recognizers from the view and the offline mode is turned off how will we restore the view's original behavior?

Reply:
I will write a new solution where it's applicable.